### PR TITLE
feat: add testing template

### PR DESCRIPTION
**Pull request by dukenguyenxyz**
These templates were added with respect to the different tests listed in release test google doc

### CODE REVIEW

File: test/shard.yml [hunk 1]
```diff
@@ -3,5 +3,4 @@ version: 1.0.0
 
 dependencies:
   placeos:
-    github: placeos/crystal-client
-    version: ~> 2
+    path: /Users/macbook/Documents/GitHub/crystal-client
```
Comment (caspiano) [inline test/shard.yml]: artefact of your personal environment here

Comment (dukenguyenxyz) [reply test/shard.yml]: oops

**Comment by kimburgess** [comment]
Converting this to a draft while it's still WIP to prevent an accidental merge.

**Comment by kimburgess** [comment]
Collapsed this into #3 (also WIP) to dedup. Will need tidy up, but that can take place within that branch.

**Pull request merged**